CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.7.5.
+
+* Return the correct HTTP status code for badly formatted requests.
```

Comment:
`UriArgumentError` is now part of the public API, right? Should document that here, too!

Reply:
Good question. Should it become part of the public API? If we make it a private class, it will still be the same `ArgumentError` as previously, and that way it doesn't change the public API.

Reply:
Better idea. Remove this new class. Throw `ArgumentError` BUT SET THE `name` param to `requestedUri`.

Then do `on ArgumentError catch (e, stackTrace)` and check the `name` for `requestedUri`.

Right?

Reply:
I like it! Updated the code, but I found it simpler to use a generic catch and do the type check afterwards (1 code block instead of 3 very similar one).

Reply:
SGTM